community/contributing/bisecting_regressions.rst

```diff
@@ -32,20 +32,27 @@ range of commits that potentially need to be built from source and tested.
 You can find binaries of official releases, as well as alphas, betas,
 and release candidates `here <https://downloads.tuxfamily.org/godotengine/>`__.
 
-For example, if you've reported a bug against Godot 3.2, you should first try to
-reproduce the bug in Godot 3.1 (not a patch release, see below for the reason).
-If the bug doesn't occur there, try to reproduce it in Godot 3.2 *beta 1* (which
-is roughly in the middle of all test builds available). If you can't reproduce
-the bug with Godot 3.2 beta 1, then try newer betas and RC builds. If you do
-manage to reproduce the bug with Godot 3.2 beta 1, then try older alpha builds.
+If you can reproduce an issue on Godot 4.0, we recommend trying to reproduce the
+issue in the latest Godot 3.x version (if the feature exhibiting the bug is
+present in 3.x). This can be used to check whether the issue is a regression in
+4.0 or not.
```

Comment:
```suggestion
If you can reproduce an issue with Godot 4.0, we recommend trying to reproduce the
issue in the latest Godot 3.x version (if the feature exhibiting the bug is
present in 3.x). This can be used to check whether the issue is a regression in
4.0 or not.
```

I'm also wondering if this is going to remain a good instruction once we release 4.0. It doesn't really cover 4.x releases, and it shouldn't be required, nor recommended IMO, to try 3.x to see if it's a regression. We'll have a lot of new users, hopefully, and making them downgrade just to check doesn't sound reasonable.

Reply:
Maybe we can reword it like this?

```
If you have experience with Godot 3.x and can reproduce an issue with Godot 4.0, we recommend trying to reproduce the issue in the latest Godot 3.x version (if the feature exhibiting the bug is present in 3.x). This can be used to check whether the issue is a regression in
4.0 or not.
```

We can remove this paragraph once 4.1 is released.

Reply:
Sure, that would be an improvement. I guess we can reintroduce a version of what this paragraph used to be when we have at least the 4.1 release, so we can refer to it in the example.